R/add_Layer.R

```diff
@@ -178,22 +244,31 @@ buildWatermarkGrob <- function(label, x = .5, y = .5, angle = 30,
 addXscale <- function(plotObject,
                       xscale,
                       xscale.args = list()) {
-  checkmate::assertChoice(xscale, choices = c("linear", "log"), null.ok = TRUE)
+  checkmate::assertChoice(xscale, choices = c("linear", "log", "discrete"), null.ok = TRUE)
```

Comment:
Possible scales should be defined as **enum**, then `validateEnumValue `from ospuite.utils can be used

Reply:
I can define scales as enum, but

-  I prefer checkmate to validateEnumValue
- In this case you have to be careful, for some cases (like timeprofiles)  only continuous x-scales like "linear" and "log"  make sense other plotTypes l(like boxwhisker) accept also discrete

Reply:
you can still use the checkmate (or whatsoever other validation) with enums and check for concrete values, e.g.
`checkmate::assertChoice(xscale, choices = c(xScales$linear, xScales$log), null.ok = TRUE)`